src/config/configuration.ts

```diff
@@ -0,0 +1,29 @@
+import { registerAs } from '@nestjs/config';
+import { IAppConfig } from '../../@types/env';
```

Comment:
It's not a good idea to import something outside the `src` boundary.
It's better to keep relevant information as close as possible and move `IAppConfig` into the config folder.